apps/nextjs/next.config.js

```diff
@@ -1,13 +1,10 @@
 import { fileURLToPath } from "url";
-import { createJiti } from "jiti";
+import createJiti from "jiti";
 
-// Import env files to validate at build time. Use jiti so we can load .ts files in here.
-await createJiti(fileURLToPath(import.meta.url)).import("./src/env");
+createJiti(fileURLToPath(import.meta.url))("./src/env");
```

Comment:
I think we don't need jiti anymore. (You can use next.config.ts directly)

Reply:
https://github.com/t3-oss/create-t3-turbo/pull/1233#discussion_r1818190991